web/angular-wallet/src/app/main/send-burst/send-burst-form/send-burst-form.component.spec.ts

```diff
@@ -1,17 +1,17 @@
 import {async, ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {NetworkModule} from 'app/network/network.module';
-import {
-  MatCheckboxModule,
-  MatGridListModule,
-  MatIconModule,
-  MatButtonModule,
-  MatInputModule,
-  MatFormFieldModule,
-  MatCardModule,
-  MatSelectModule,
-  MatProgressSpinnerModule, MatTooltipModule, MatDialogModule
-} from '@angular/material';
+import { MatButtonModule } from '@angular/material/button';
```

Comment:
is there a true reason, why this is better than the former import (tree-shaking?)